scripts/queueTransaction.js

```diff
@@ -0,0 +1,96 @@
+const config = require("./config/deploymentConfig");
+const deployedAddresses = require("./config/deployedAddresses");
+const thorify = require("thorify").thorify;
+const Web3 = require("web3");
+const Timelock = require(config.pathToTimelockJson);
+const GovernorAlpha = require(config.pathToGovernorAlphaJson);
+const fs = require('fs');
+const assert = require('assert');
+
+let rpcUrl = null;
+if (process.argv.length < 3) 
+{
+    console.error("Please specify network, either mainnet or testnet");
+    process.exit(1);
+} 
+else
+{
+    if (process.argv[2] == "mainnet") rpcUrl = config.mainnetRpcUrl;
+    else if (process.argv[2] == "testnet") rpcUrl = config.testnetRpcUrl;
+    else {
+        console.error("Invalid network specified");
+        process.exit(1);
+    }
+}
+
+const web3 = thorify(new Web3(), rpcUrl);
+
+web3.eth.accounts.wallet.add(config.privateKey);
+
+queueTransaction = async() =>
```

Comment:
Should file / function be renamed to `queueTimelockTransfer`?

Reply:
Yes makes sense, changed both names.